pyteal/ast/txn.py

```diff
@@ -28,6 +28,7 @@ class TxnType:
     AssetTransfer = EnumInt("axfer")
     AssetFreeze = EnumInt("afrz")
     ApplicationCall = EnumInt("appl")
+    Any = EnumInt("txn")
```

Comment:
This isn't right.

This enum matches the output of `txn TypeEnum`, which is an int. Furthermore, it uses the "enum" int ops to load these values to make it more obvious, e.g. `int acfg` will translate to the right integer that matches the `txn TypeEnum` value.

So an "any" value here doesn't make sense for two reasons: `int any` won't work, and `txn TypeEnum` will never return a value for "any"

Reply:
I added this here so I could re-use the existing txn types and because we have an `Unknown` type already. I'll redefine them in the abi/transaction.py though for the reasons you mentioned